partials/content-toc.php

```diff
@@ -46,8 +56,15 @@
 		</li>
 	</ol><!-- end #toc -->
 
-	<?php $output = ob_get_clean();
-	echo $output;
-	set_transient( $toc_transient, $output );
+	<?php
+	$toc_output = ob_get_clean();
+	set_transient( $toc_transient, $toc_output );
 }
 
+// Search for [ id="toc-chapter-123" class=" ] and replace (once!) with [ id="toc-chapter-123" class="toc__selected  ]
+if ( isset( $post ) ) {
```

Comment:
This looks great. One other case we need to handle:

```
if ( isset( $post ) ) {}

Reply:
Cool. It already handles part, front-matter, back-matter... The comment is misleading.

Reply:
Right, just to be clear, if I have: